docs/atomic-run.1.md

```diff
@@ -49,6 +49,9 @@ These defaults are suggested values for your container images.
 **SUDO_GID**
   The `SUDO_GID` environment variable.  This is useful with the docker `-u` option for user space tools.  If the environment variable is not available, the default GID of the value for `SUDO_UID` is used.  If this value is not available, the value of `/proc/self/loginuid` is used.
 
+**RUN_OPTS**
```

Comment:
How about we just call this OPTS_FILE and then we can use it for INSTALL, RUN, UNINSTALL. Any reason to have multiple?